pylucene/app/adapter.py

```diff
@@ -128,6 +132,7 @@ def index_issues(self,
             doc.add(Field('summary', issue.summary, StoredField.TYPE))
             doc.add(Field('description', issue.description, StoredField.TYPE))
             doc.add(Field('text', f'{issue.summary}. {issue.description}', TextField.TYPE_STORED))
+            doc.add(Field('comments',f'{issue.comments}', TextField.TYPE_STORED))
```

Comment:
View problems with this. 
1) You're adding the string representation of the python object holding the comments, so you might be introducing additional text
2) This indexes the comments as a single document, not separately. You might want to discuss with Mohamed what the intended goal would be here.